.travis.yml

```diff
@@ -1,76 +1,36 @@
 env:
   global:
-    - JAVA_HOME=/usr/lib/jvm/java-8-oracle
-    - APP_VERSION=$TRAVIS_TAG
     - PACKAGE_NAME=org.phidatalab.radar_armt
```

Comment:
Perhaps this can be `PACKAGE_NAME_ANDROID`?

Reply:
Made the changes